pydynamo_brain/pydynamo_brain/ui/radiiPainter.py

```diff
@@ -210,3 +253,15 @@ def zoomedLocation(self, xyz):
     # HACK - utilities
     def isNearZ(self, z):
         return abs(z - self.zAt) < 3
+
+    #Given a selected point and an additional neightboring point
+    #returns a point perpendicular to the line joining these points
+    def caliperPoints(self, origin, point, ifNext):
+        origin = origin.location
```

Comment:
I'd rename the variables - it's weird that origin changes data type here.
or better yet, just `x, y, _ = util.normDelta(point.location, origin.location)`